packages/storybook/stories/CallComposite/BasicExample.tsx

```diff
@@ -29,7 +30,7 @@ export const BasicExample: () => JSX.Element = () => {
   }, [knobs]);
 
   return (
-    <div style={COMPOSITE_EXPERIENCE_CONTAINER_STYLE}>
+    <Stack horizontalAlign="center" verticalAlign="center" styles={compositeExperienceContainerStyle}>
```

Comment:
If you're replacing `div` with `Stack`, shouldn't you also wrap the contents with `Stack.Item`?

Reply:
As far as I understand, each child is essentially a Stack.Item, the reason to use a Stack.Item is if you want to apply certain properties to different children (e.g. if different items grow at different rates relative to each other).